source/management/identity-types.yml

```diff
@@ -7,8 +7,8 @@ paths:
       description: |
         Retrieves a list of identity types; supports filtering, sorting, and pagination. Requires admin access.
       security:
-        - ztSession: []
```

Comment:
Looks like annoying auto-reformatting. We are probably using two different plugins. I'm using one for Open API you might have one for YAML, and they are using their defaults to fight.